recipes/dj-static/meta.yaml

```diff
@@ -0,0 +1,40 @@
+{% set package = "dj-static" %}
```

Comment:
@kwilcox - I completely agree. If you are happy to maintain your recipe in this way, I'm :+1: